docs/site_info/rap-release-workflow.md

```diff
@@ -7,6 +7,32 @@ tags:
 
 #
 
+## Current Release Process (coding in the open)
+
+!!! warning "The release process below is no longer in use!"
```

Comment:
Your call but consider moving this warning below the "Old release process" header, when I first looked at this, I thought it referred to the new content